Uno.Gallery/Uno.Gallery.Shared/Uno.Gallery.Shared.projitems

```diff
@@ -64,7 +64,6 @@
     <Content Include="$(MSBuildThisFileDirectory)Assets\Fonts\Material\Roboto-Light.ttf" />
     <Content Include="$(MSBuildThisFileDirectory)Assets\Fonts\Material\Roboto-Medium.ttf" />
     <Content Include="$(MSBuildThisFileDirectory)Assets\Fonts\Material\Roboto-Regular.ttf" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Fonts\uno-fluentui-assets.ttf" />
```

Comment:
should delete that file too

Reply:
https://github.com/unoplatform/Uno.Gallery/blob/dev/mara/update-icon-font/Uno.Gallery/Uno.Gallery.Shared/Assets/Fonts/uno-fluentui-assets.ttf

Reply:
Missed that one thanks! Search doesn't bring up any other results